tool/migrate_nook_watcher_data.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+import textwrap
+
+import firebase_util
+from core_data_modules.logging import Logger
+
+firebase_client = None
+log = None
+
+def strip_project(data):
+    for entry in data:
+        try:
+            del list(entry.values())[0]['project']
+        except Exception:
+            pass
+    return data
+
+def migrate():
+    if args.project:
+        docs = firebase_client.collection(args.source).where('project', '==', args.project).stream()
+    else:
+        docs = firebase_client.collection(args.source).stream()
+    data = [{doc.id: doc.to_dict()} for doc in docs][-20:]
```

Comment:
This seems to be selecting only the last 20 docs, which doesn't seem right to me? is it an artefact from testing or am I misunderstanding something?

Reply:
It's from testing. Removed it